docs/building-spark.md

```diff
@@ -259,6 +259,14 @@ or
 Java 8 tests are automatically enabled when a Java 8 JDK is detected.
 If you have JDK 8 installed but it is not the system default, you can set JAVA_HOME to point to JDK 8 before running the tests.
 
+## PySpark pip installable
+
+If you are building Spark for use in a Python environment and you wish to pip install it, you will first need to build the Spark JARs as described above. Then you can construct an sdist package suitable for setup.py and pip installable package.
+
+    cd python; python setup.py sdist
```

Comment:
So `make-distribution.sh` will copy the files into the distribution directory. If you just want to test the pip install part `./build/sbt package;cd python; python setup.py sdist; cd dist; pip install *.tar.gz` should do the trick.